File: Cap 04/Topico 4.3/Adicao de Numeros 1/adcnum1.pas
{*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 * 
 * Capitulo ..: 4 - Topico 4.3
 * 
 * Programa ..: ADICAO DE NÚMEROS - DECISAO SIMPLES
 *}

program ADICAO_DE_NUMEROS;
var
  A, B, X: Real;
begin
  WriteLn('DECISAO SIMPLES - ADICAO DE DOIS NUMEROS');
  WriteLn;
  Write('Entre valor <A>: ');
  ReadLn(A);
  Write('Entre valor <B>: ');
  ReadLn(B);
  X := A + B;
  if (X > 10) then
  begin
    WriteLn('Resultado da adicao quando maior que dez = ', X:6:2);
  end;
end.

File: Cap 04/Topico 4.3/Adicao de Numeros 1/adcnum1.js
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 * 
 * Capitulo ..: 4 - Topico 4.Programacao
 * 
 * Programa ..: ADICAO DE NUMEROS - DECISAO SIMPLES
 */

console.log('DECISAO SIMPLES - ADICAO DE DOIS NUMEROS');
console.log();
  
const A = parseFloat(prompt('Entre valor <A>: '));
const B = parseFloat(prompt('Entre valor <B>: '));

const X = A + B;

if (X > 10) {
  console.log('Resultado da adicao quando maior que dez = ', X.toFixed(2));
}
